### PR TITLE
New About page

### PR DESCRIPTION
**Pull request by marik22312**

### CODE REVIEW

File: pages/about/index.tsx [hunk 1]
```diff
@@ -1,93 +1,84 @@
...
               </p>
             </div>
-              <div className={styles.storyImageWrapper}>
-                <img
-                  src="https://res.cloudinary.com/marik-shnitman/image/upload/f_auto,dpr_auto,q_auto/v1606921730/radiosavta/gallery/radiosavta3"
-                  alt="Our team image"
-                  className={styles.backgroundImage}
+          </div>
+          <div className={style.pateronCta}>
+            <h3>תמכו באנו והפכו לפטרונים!</h3>
```
Comment (asaf147369) [inline pages/about/index.tsx:57]: באנו?

Comment (marik22312) [reply pages/about/index.tsx:57]: WIP LOL

File: pages/about/AboutPageNew.module.scss [hunk 2]
```diff
@@ -0,0 +1,160 @@
+@import "../../styles/mixinsNew";
+
+.pageWrapper {
+  font-size: 1rem;
+  padding-top: 4rem;
+
+  @include media-breakpoint(md) {
```
Comment (asaf147369) [inline pages/about/AboutPageNew.module.scss:7]: Why need same padding in media query?

File: pages/about/AboutPageNew.module.scss [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+  font-size: 0.7rem;
+
+  @include media-breakpoint(md) {
+    margin-top: var(--SPACE1);
+    padding: 10px 20px;
+  }
+}
+
+.teamSection {
+  padding: var(--SPACE3) var(--SPACE3);
```
Comment (asaf147369) [inline pages/about/AboutPageNew.module.scss:143]: Write just 1 time

File: pages/about/AboutPageNew.module.scss [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+  background-color: var(--dark-green);
+  @include media-breakpoint(md) {
+    padding: var(--SPACE3) 0;
+  }
```
Comment (asaf147369) [inline pages/about/AboutPageNew.module.scss:147]: `padding-inline: 0;`

File: pages/about/AboutPageNew.module.scss [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+    padding-top: 4rem;
+  }
+}
+
+.aboutUsSection {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+
+  background-image: url("https://res.cloudinary.com/marik-shnitman/image/upload/v1643729419/radiosavta/backgrounds/about_mobile_desktop.png");
```
Comment (marik22312) [inline pages/about/AboutPageNew.module.scss:17]:
```suggestion
  background-image: url("https://res.cloudinary.com/marik-shnitman/image/upload/f_auto,dpr_auto,q_auto/v1643729419/radiosavta/backgrounds/about_mobile_desktop.png");
```

File: pages/about/AboutPageNew.module.scss [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+  background-repeat: no-repeat;
+  background-position-y: calc(var(--navbar-width) - 60px);
+
+  @include media-breakpoint(md) {
+    background-attachment: fixed;
+    background-image: url("https://res.cloudinary.com/marik-shnitman/image/upload/v1643729581/radiosavta/backgrounds/about_background_deskyop.png");
```
Comment (marik22312) [inline pages/about/AboutPageNew.module.scss:23]:
```suggestion
    background-image: url("https://res.cloudinary.com/marik-shnitman/image/upload/f_auto,dpr_auto,q_auto/v1643729581/radiosavta/backgrounds/about_background_deskyop.png");
```

File: pages/about/index.tsx [hunk 7]
```diff
@@ -1,96 +1,120 @@
...
+              </h1>
+            </div>
+            <div className={style.aboutUsText}>
               <p>
-                קולקטיב 'רדיו סבתא' נוסד בשנת 2015 כתחנת רדיו אינטרנטי במצפה
-                רמון. התחנה נוסדה ע"י עירא דיין, במטרה להנציח את (סבתא) יעל קרן,
-                חותנתו של עירא
+                קולקטיב &apos;רדיו סבתא&apos; נוסד בשנת 2015 כתחנת רדיו אינטרנטי
+                במצפה רמון. התחנה נוסדה ע&quot;י עירא דיין, במטרה להנציח את
+                (סבתא) יעל קרן, חותנתו של עירא
```
Comment (marik22312) [inline pages/about/index.tsx:32]:
```suggestion
                (סבתא) יעל קרן, חמתו של עירא
```

**Pull request merged**